src/pages/Blog.js

```diff
@@ -1,11 +1,182 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useContext } from "react";
 import Container from "@material-ui/core/Container";
+import { useParams } from "react-router-dom";
+import { loadBlogPost } from "../data/feedLibrary";
+import { SkynetContext } from "../state/SkynetContext";
+import ReactMarkdown from "react-markdown";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import CardHeader from "@material-ui/core/CardHeader";
+import Avatar from "@material-ui/core/Avatar";
+import IconButton from "@material-ui/core/IconButton";
+import Twitter from "@material-ui/icons/Twitter";
+import GitHub from "@material-ui/icons/GitHub";
+import Telegram from "@material-ui/icons/Telegram";
+import Skeleton from "@material-ui/lab/Skeleton";
+import { displayName } from "../data/displayName";
+import { useHistory } from "react-router-dom";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import { makeStyles } from "@material-ui/core/styles";
+
+//style for removing hover shading
+const useStyles = makeStyles((theme) => ({
+  focus: {
+    "&&&": {
+      opacity: 0,
+    },
+  },
+}));
 
 //Blog page component, used to create view blog posts, returns JSX layout for rendering blogs
 export default function Blog(props) {
+  const { getUserProfile, feedDAC } = useContext(SkynetContext); //values to use from Skynet context
```

Comment:
Follow up: we typically prefer comments as their own line. One reason is it helps keep line length down and makes it easier to read the comment prior to the code as oppose to the code then the comment.

Reply:
For blocks of code like this, where you are defining a bunch of variables/state, we sometimes will combine the comments into a single block like:
```
// Comment block
//
// Thing 1
//
// Thing 2
// ...
Thing 1 code
Thing 2 code 
...
```